content/blog/2017/2017-02-14-pipeline-editor-preview.adoc

```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+title: "Say 'Hello' to the Blue Ocean Pipeline Editor"
+tags:
+- blueocean
+- editor
+- declarative
+- pipeline
+author: jamesdumay
+---
+
+link:/blog/2016/09/19/blueocean-beta-declarative-pipeline-pipeline-editor/[Back in September 2016 we announced the availability of the Blue Ocean beta]
+and the forthcoming Visual Pipeline Editor. We are happy to announce that you can try
+the Pipeline Editor preview release today.
+
+=== What is it?
+
+The Visual Pipeline Editor is the simplest way for anyone wanting to get started with
+creating Pipelines in Jenkins. It's also a great way for advanced Jenkins users
+to start adopting pipeline. It allows developers to break up their pipeline into different
+ stages and parallelize tasks that can occur at the same time - graphically.
+ The rest is up to you.
+
+Jenkinsfiles belong in the Git repository where they are versioned with 
+your application code.
+
+A pipeline you create visually will produce a link://blog/2017/02/03/declarative-pipeline-ga/[Declarative Pipeline] Jenkinsfile for you. This is a "round trip" editor which lets you load the pipeline back in later, for more editing. Not sure what a Jenkins Pipeline is? Why not check out our link:/doc/[new guided tour]!
+
+image::/images/blueocean/editor-1.png["The Editor", role="center"]
+
+=== What are we doing next?
+
+We are working hard to provide feature parity between the Declarative Pipeline syntax and the visual editor. The next phase is to integrate the editor into Blue Ocean so that you don't have to leave the UI and commit the Jenkinsfile to your repository to complete authoring your pipeline.
+
+In Blue Ocean, you will be able to edit a `Jenkinsfile`
+for a branch directly from within the user interface using the Visual Pipeline Editor. When you are done authoring your pipeline, the pipeline definition will be saved back to your repository as a `Jenkinsfile`. You can edit the Pipeline again using the Visual Editor or from your favorite text editor.
+
+We are hoping to deliver this level of integration into Blue Ocean and the
+Visual Pipeline Editor over the next few months, so be sure to check regularly for updates in
+the Jenkins plugin manager.
+
+=== Get the Preview
+
+The Visual Pipeline Editor is available in preview today.
+
+To try it out today:
+
+. Install the `Blue Ocean Pipeline Editor` from the Jenkins plugin manager
+. Click on the `Open Blue Ocean` button and then the `Pipeline Editor` in the main navigation
+
+We are looking forward to your feedback to help make the Editor
```

Comment:
Visual Pipeline Editor
Pipeline Editor
Editor

that last one is perhaps an unnecessary variant?